configs/98netflix

```diff
@@ -0,0 +1,6 @@
+#!/system/bin/sh
```

Comment:
no, i don't care whatever the hell does it fix but it only apply to 3rd app but have nothing related to functional of ROM